pages/src/navigation/sidebar/sidebar.less

```diff
@@ -7,9 +7,9 @@ esl-d-sidebar {
   background: @nav-dark-bg;
 
   position: fixed;
+  top: 56px;
   z-index: 10;
-  left: 0;
-  height: 100vh;
+  height: calc(100vh - 56px);
```

Comment:
For cases like this, it is better to define the value of 56px in a variable and use the variable in your code. This makes it easier to understand that your top position value and height value are related to each other.